src/compiler/crystal/command/format.cr

```diff
@@ -173,7 +173,7 @@ class Crystal::Command
       if check_files
         check_files << FormatResult.new(filename, FormatResult::Code::SYNTAX)
       else
-        STDERR << "Syntax Error:".colorize(:yellow).toggle(@color) << " " << ex.message << " at " << filename << ":" << ex.line_number << ":" << ex.column_number << "\n"
+        STDERR << "Syntax Error:".colorize(:yellow).toggle(@color) << " " << ex.message << " at " << filename << ":" << ex.line_number << ":" << ex.column_number << '\n'
```

Comment:
You are missing some `" "` & `":"` here

Reply:
Yep, there are still some missing occurrences I intentionally left untouched due to compiler errors, which I didn't had time to debug, yet.